doc/api/events.md

```diff
@@ -155,6 +155,18 @@ myEmitter.emit('error', new Error('whoops!'));
 // Prints: whoops! there was an error
 ```
 
+It is possible to monitor `'error'` events without consuming the emitted error
+by installing a listener using the symbol `errorMonitorSymbol`.
+
+```js
+const myEmitter = new MyEmitter();
+myEmitter.on(errorMonitorSymbol, (err) => {
```

Comment:
Nit: maybe show how to access `errorMonitorSymbol` here.

I did see that it's not the case for this doc but we mostly try to keep all examples fully encapsulated and possible to run on their own without any code change necessary.